src/components/atoms/_OcGhostElement/_OcGhostElement.vue

```diff
@@ -1,6 +1,22 @@
 <template>
   <div id="ghost-element" class="ghost-element">
-    <oc-icon :name="firstPreviewIcon" size="large" />
+    <div class="icon-wrapper">
+      <oc-resource-icon
+        v-if="iconFolder"
+        class="ghost-element-icon ghost-element-icon-folder"
+        :resource="iconFolder"
+      />
+      <div v-for="(item, index) in iconStack" :key="index" :style="{ 'z-index': key }">
+        <oc-icon
+          :style="getStyle(index)"
+          class="ghost-element-icon-background"
+          name="resource-type-file"
+          size="large"
+          variation="inverse"
+        />
```

Comment:
Any idea for improvement welcome: This is a file icon without any cutout rendered behind the actual file icon. This prevents it from looking weird, look for example at the image or text icon. They both have a cutout in the middle which would look bad in this case.